windows/progress-view/ProgressViewView.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "JSValueXaml.h"
+#include "ProgressViewView.h"
+#include "ProgressViewView.g.cpp"
+
+namespace winrt {
+    using namespace Microsoft::ReactNative;
+    using namespace Windows::Foundation;
+    using namespace Windows::Foundation::Metadata;
+    using namespace Windows::UI;
+    using namespace Windows::UI::Xaml;
+    using namespace Windows::UI::Xaml::Controls;
+    using namespace Windows::UI::Xaml::Input;
+    using namespace Windows::UI::Xaml::Media;
+    using namespace Windows::UI::Xaml::Media::Imaging;
+} // namespace winrt
+
+namespace winrt::progress_view::implementation {
+
+    ProgressViewView::ProgressViewView(winrt::IReactContext const& reactContext) : m_reactContext(reactContext) {
+        this->Minimum(0);
+        this->Maximum(100);
+    }
+
+    void ProgressViewView::UpdateProperties(winrt::IJSValueReader const& reader) {
+        auto const& propertyMap = JSValueObject::ReadFrom(reader);
+
+        for (auto const& pair : propertyMap) {
+            auto const& propertyName = pair.first;
+            auto const& propertyValue = pair.second;
+
+            if (propertyName == "progress") {
+                this->Value(propertyValue.AsDouble() * 100);
+            }
+            else if (propertyName == "progressTintColor") {
+                this->Foreground(propertyValue.To<winrt::Brush>());
+
+            }
+            else if (propertyName == "trackTintColor") {
+                this->Background(propertyValue.To<winrt::Brush>());
+            }
+            else if (propertyName == "progressViewStyle") {
+                if (propertyValue == "bar") {
+                    this->Background(SolidColorBrush(ColorHelper::FromArgb(000, 000, 000, 000)));
```

Comment:
I'm using the same class right now and it doesn't compile without #include "winrt/Windows.UI.Xaml.Media.h" - interesting you don't need this here

Reply:
It's in my pch.h file, would it be easy to read if I moved it to this file?

Reply:
oh, I didn't notice! I'm not sure if you need to take it out of there. It does seem like pch.h gets included in all cpp files, which might not be necessary.